test/unittests/commands/excel2json/test_models_json_header.py

```diff
@@ -0,0 +1,145 @@
+import pytest
+
+from dsp_tools.commands.excel2json.models.json_header import Description
+from dsp_tools.commands.excel2json.models.json_header import Descriptions
+from dsp_tools.commands.excel2json.models.json_header import ExcelJsonHeader
+from dsp_tools.commands.excel2json.models.json_header import Keywords
+from dsp_tools.commands.excel2json.models.json_header import Prefixes
+from dsp_tools.commands.excel2json.models.json_header import Project
+from dsp_tools.commands.excel2json.models.json_header import User
+from dsp_tools.commands.excel2json.models.json_header import Users
+
+SCHEMA = "https://raw.githubusercontent.com/dasch-swiss/dsp-tools/main/src/dsp_tools/resources/schema/project.json"
+
+
+@pytest.fixture()
+def prefixes() -> Prefixes:
+    return Prefixes({"dcelements": "http://purl.org/dc/elements/1.1/"})
+
+
+@pytest.fixture()
+def description_de() -> Description:
+    return Description("de", "Beschreibungstext")
+
+
+@pytest.fixture()
+def description_en() -> Description:
+    return Description("en", "description text")
+
+
+@pytest.fixture()
+def descriptions(description_de: Description, description_en: Description) -> Descriptions:
+    return Descriptions([description_de, description_en])
+
+
+@pytest.fixture()
+def keywords() -> Keywords:
+    return Keywords(["Keyword 1"])
+
+
+@pytest.fixture()
+def user_sys_admin() -> User:
+    return User("sys_admin", "sys_admin@email.ch", "given name1", "family name1", "PW1", "en", sys_admin=True)
+
+
+@pytest.fixture()
+def user_member() -> User:
+    return User("member", "member@email.ch", "given name2", "family name2", "PW2", "de", member=True)
+
+
+@pytest.fixture()
+def user_admin() -> User:
+    return User("admin", "admin@email.ch", "given name3", "family name3", "PW3", "de", admin=True)
+
+
+@pytest.fixture()
+def users(user_sys_admin: User, user_member: User, user_admin: User) -> Users:
```

Comment:
This is the only place where the single users are used. So you could consider removing the singe-user-fixtures, and instead create them inside this fixture.

Reply:
This happened because of future optimisation. So that we can mix and match. But I can change it